README.md

```diff
@@ -54,6 +54,7 @@
 - [jest-runner-prettier](https://github.com/keplersj/jest-runner-prettier) Prettier runner for Jest.
 - [jest-electron-runner](https://github.com/d4rkr00t/jest-electron-runner) Electron runner for Jest.
 - [jest-runner-stylelint](https://github.com/keplersj/jest-runner-stylelint) Stylelint runner for Jest.
+- [jest-electron](https://github.com/hustcc/jest-electron) Easy way to run and debug jest unittest in electron.
```

Comment:
I wonder how is it different from `jest-electron-runner` seen above?

Reply:
1. **debug mode**, can debug the case in electron, include refresh browser to re-run, e.g. when we test the vis chart https://github.com/facebook-atom/jest-electron-runner/issues/50
2. multi-renderer to run case, higher performance https://github.com/hustcc/jest-electron/pull/1

The first is the main pain point, so rewrite it.

Reply:
what do you think @rickhanlonii?

Reply:
we will migrate the unit test from torchjs which is mocha + electron to jest in [antvis](https://github.com/antvis).

The debug mode is necessary.

Reply:
Cool! It would be just nice to have a single way to test Electron, maybe there's some space for collaboration with folks at jest-electron-runner :)

Reply:
The issue does not  be reply, a pr with lot of modified, I am not sure will accepted.

In addition, the code is too obscure, i am not sure that i have the ability to add the e features.

Reply:
Maybe they'll be good to recommend your library if it's better, who knows!

Reply:
So what can i do？

Reply:
Wait please :)

Reply:
What a joke~:joy: